statslive.php

```diff
@@ -72,7 +72,7 @@ private function getCustomersOnline()
 					INNER JOIN `'._DB_PREFIX_.'customer` u ON u.id_customer = g.id_customer
 					WHERE cp.`time_end` IS NULL
 						'.Shop::addSqlRestriction(false, 'c').'
-						AND TIME_TO_SEC(TIMEDIFF(\''.pSQL(date('Y-m-d H:i:00', time())).'\', cp.`time_start`)) < 900
+						AND `cp.time_start` > NOW() - interval 900 second
```

Comment:
Just for history's sake, there's a small SQL error with backticks on both modified lines.

This will be fixed shortly in a follow-up PR

Reply:
Fixed in #5